feature-toggles/feature-toggles-api/src/main/java/com/duckduckgo/feature/toggles/api/FeatureToggles.kt

```diff
@@ -108,14 +110,35 @@ class FeatureToggles private constructor(
 }
 
 interface Toggle {
+    /**
+     * This is the method that SHALL be called to get whether a feature is enabled or not. DO NOT USE [getRawStoredState] for that
+     * @return `true` if the feature should be enabled, `false` otherwise
+     */
     fun isEnabled(): Boolean
 
+    /**
+     * The usage of this API is only useful for internal/dev settings/features
+     * If you find yourself having to call this method in production code, then YOUR DOING SOMETHING WRONG
+     *
+     * @param state update the stored [State] of the feature flag
+     */
     fun setEnabled(state: State)
 
+    /**
+     * The usage of this API is only useful for internal/dev settings/features
+     * If you find yourself having to call this method in production code, then YOUR DOING SOMETHING WRONG
+     *
+     * @return the raw [State] store for this feature flag.
```

Comment:
Can you add a line explaining what the raw state means? I understand that `isEnabled` is the important API here generally, but i'm not sure what raw state really is from reading this.

Reply:
Done